res/database/Default/database_programmes_fictional.xml

```diff
@@ -1178,60 +1178,79 @@ Do the two offer enough material for an exciting panel?</en>
 				</programme>
 			</children>
 		</programme>
-		<programme id="9ad21f32-6e2d-45c9-b3c2-7d758fa30d25" product="7" licence_type="3" fictional="1" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny" comment="TODO: Shouldn't this be a show to produce?">
+		<programme id="9ad21f32-6e2d-45c9-b3c2-7d758fa30d25" product="7" licence_type="3" fictional="1" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny" comment="modified by scr0llbaer. Created live script version d6242a49-71e0-4997-97d6-3684e5c67d54">
 			<title>
-				<de>Dynamit - Das Magazin</de>
-				<en>Dynamite - The Magazine</en>
+				<de>Dynamit - Das Magazin Spezial: [2|Nick] und [3|Nick]</de>
+				<en>Dynamite - The Magazine Special: [2|Nick] and [3|Nick]</en>
 			</title>
 			<description>
-				<de>Das Ex-Model [1|Full] präsentiert aktuellen Tratsch, Promi-Geflüster und tränenrührende Tagesgeschehnisse.
-
-Ein Muss für jede Hausfrau, ein Fluch für harte Männer.</de>
-				<en>The ex-model [1|Full] presents current gossip, celebrity rumors and tear-jerking daily events.
-
-A must for every housewife, a curse for hard men.</en>
+				<de>Das Ex-Model [1|Full] präsentiert die Spezialausgabe unseres Dynamitmagazins anlässlich des 30. Geburtstages von [3|Nick].</de>
+				<en>The ex-model [1|Full] presents the special edition of our dynamite magazine on the occasion of [3|Nick]'s 30th birthday.</en>
 			</description>
 			<staff>
 				<member index="0" function="1">cebd1a68-cf97-4b93-b3f9-ac34bda4bdcc</member>
 				<member index="1" function="2">88ba23dc-d4f1-4c77-9ddf-8dbf4a4ce3b2</member>
+				<member index="2" function="64">28fe86a0-74ca-43a7-bc49-d3bd4df7c4d3</member>
+				<member index="3" function="64">4623935a-750a-4e5a-9d1b-6875062ff2a8</member>
 			</staff>
 			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data country="D" year="1991" distribution="2" maingenre="301" subgenre="" flags="0" blocks="1" price_mod="1" />
+			<data country="D" distribution="2" maingenre="301" subgenre="" flags="0" blocks="1" price_mod="1" />
```

Comment:
If you remove the year:
- consider replacing with a relative year (`year="2"` would make it appear ingame 2 years after game start, "-2" would make it a production of 2 yrs ago)
- the "30. Geburtstag" might of course no longer fit ... depending on how "near to reality" you want to be, this won't work.

Reply:
I didn't remove the year, I had to change it to `releasetime` because I had to add `day="7"`